README.md

```diff
@@ -42,6 +42,10 @@ ink-wrapper-types = { git = "https://github.com/Cardinal-Cryptography/ink-wrappe
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
 ink_primitives = "4.0.1"
 
+# You only need this one if you have messages of the form `Trait::message`, like the ones generated by openbrush, for
+# example.
+async-trait = "0.1.68"
```

Comment:
Don't you need it always? All contract interaction is async.

Reply:
"Regular" contract methods (not coming from traits) are not generated as trait methods, but in `impl Instance`, so they can just be async without any extra effort. This is only for the `#[async_trait]` on `trait PSP22`, etc.

Reply:
Ah, true.